packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/utils.ts

```diff
@@ -19,17 +19,11 @@ export const convertEthToUsd = (
   return (eth * rate).toFixed(2);
 };
 
-export const buildEtherscanLink = (ethChainId: number, txHash: string) => {
-  switch (ethChainId) {
-    case 81457:
-      return `https://blastscan.io/tx/${txHash}`;
-    case 84532:
-      return `https://sepolia.basescan.org/tx/${txHash}`;
-    case 11155111:
-      return `https://sepolia.etherscan.io/tx/${txHash}`;
-    default:
-      return `https://basescan.org/tx/${txHash}`;
-  }
+export const buildEtherscanLink = (txHash: string) => {
+  //const prefix = 'Base.';
+  const url =
+    app.chain.meta.ChainNode.block_explorer ?? 'https://basescan.org/';
```

Comment:
You should optionally chain like:
`app.chain?.meta?.ChainNode?.block_explorer`

Because due to the way we load meta (via some hook), it could be in a re-render.